src/libraries/System.Collections.Specialized/tests/ListDictionaryTests.cs

```diff
@@ -137,7 +137,7 @@ public static void Set_CustomComparerTest()
 
             int visited = s_dictionaryData.Length;
 
-            Assert.All(s_dictionaryData.Reverse(), element =>
+            Assert.All(Enumerable.Reverse(s_dictionaryData), element =>
```

Comment:
Why does this one need to change? We're only building here for netcoreappcurrent, which should be able to see the array-based overload, right?

Reply:
Oh, are these all in the same boat as your comment on the immutable tests, that they're not seeing the new Enumerable overload because the tests themselves are in the system namespace?